internal/compiler/passes/infer_aliases_types.rs

```diff
@@ -116,7 +116,7 @@ fn resolve_alias(
             let nr = nr.unwrap();
             let purity = nr.element().borrow().lookup_property(nr.name()).declared_pure;
             let mut elem = elem.borrow_mut();
-            let mut decl = elem.property_declarations.get_mut(prop).unwrap();
+            let decl = elem.property_declarations.get_mut(prop).unwrap();
```

Comment:
I wonder why that this was not cought before. and didn't show on the CI that denies warnings